src/DataHandler.cpp

```diff
@@ -33,3 +34,38 @@ void DataHandler::InstallHooks()
 	DataHandlerCTORHook::Install();
 #endif
 }
+
+SkyrimVRESLPluginAPI::SkyrimVRESLInterface001 g_interface001;
+
+// Constructs and returns an API of the revision number requested
+void* GetApi(unsigned int revisionNumber)
+{
+	switch (revisionNumber) {
+	case 1:
+		logger::info("Interface revision 1 requested");
+		return &g_interface001;
+	}
+	return nullptr;
+}
+
+// Handles skse mod messages requesting to fetch API functions
+void SkyrimVRESLPluginAPI::ModMessageHandler(SKSE::MessagingInterface::Message* message)
+{
+	if (message->type == SkyrimVRESLMessage::kMessage_GetInterface) {
+		SkyrimVRESLMessage* modmappermessage = (SkyrimVRESLMessage*)message->data;
+		modmappermessage->GetApiFunction = GetApi;
+		logger::info("Provided SkyrimVRESL plugin interface to {}", message->sender);
+	}
+}
+
+// Fetches the SkyrimVRESL version number
+unsigned int SkyrimVRESLPluginAPI::SkyrimVRESLInterface001::GetBuildNumber()
+{
+	return 1;  // TODO: Figure out how to get packed version number
```

Comment:
`Version::MAJOR` should work here for this, although we'd need to update `CMakeLists` and `vcpkg.json` to be `1.0.0` instead of the current version `0.0.1`

Reply:
Commonlibng had a pack function; commonlibVR doesn't. It feels like we should leverage that later. I mean it probalby doesn't matter in the short term.